src/unbound_args.jl

```diff
@@ -1,17 +1,35 @@
 """
     test_unbound_args(module::Module)
+    test_unbound_args(unbounds)
 
 Test that all methods in `module` and its submodules do not have
 unbound type parameters. An unbound type parameter is a type parameter
 with a `where`, that does not occur in the signature of some dispatch
-of the method.
+of the method. If unbounds methods are already known, they can be
+passed directly to the function instead of the module.
 
 # Keyword Arguments
 - `broken::Bool = false`: If true, it uses `@test_broken` instead of
   `@test` and shortens the error message.
+- `ignore::Vector{Tuple{Function, DataType...}}`: A list of functions and their
```

Comment:
For the ambiguity test, the analogous kwarg there is named `exclude`. I would prefer for these to be more or less aligned.